regexes/device/mobiles.yml

```diff
@@ -5206,13 +5260,21 @@ Samsung:
       model: 'GALAXY A5 (2016)'
     - regex: '(?:SAMSUNG-)?SM-A520F'
       model: 'GALAXY A5 (2017)'
+    - regex: '(?:SAMSUNG-)?SM-(?:A720F|A700[FKLS]|A710[FMY0])'
+      model: 'GALAXY A7'
     - regex: '(?:SAMSUNG-)?SM-A530F'
       model: 'GALAXY A8'
+    - regex: '(?:SAMSUNG-)?SM-A9000'
+      model: 'GALAXY A9'
+    - regex: '(?:SAMSUNG-)?SM-A910F'
+      model: 'GALAXY A9 Pro'
     - regex: '(?:SAMSUNG-)?GT-I9301I'
       model: 'GALAXY S III Neo'
     - regex: '(?:SAMSUNG-)?SM-J120[AFHMT]'
       model: 'GALAXY J1'
-    - regex: '(?:SAMSUNG-)?SM-J111[FGLM]'
+    - regex: '(?:SAMSUNG-)?SM-J110[FHML]'
+      model: 'GALAXY J1 Ace'
+    - regex: '(?:SAMSUNG-)?SM-J111F'
```

Comment:
Did you remove `J111[GLM]` on purpose?

Reply:
I don't think they actually exist: couldn't find information on those on the internet, and there were no tests/user-agents on them.

Reply:
There seems to be at least a `SM-J111M` (see https://www.handsetdetection.com/device-detection-database/devices/samsung/sm-j111m/)

Reply:
Ok, indeed, added this one back!